File: app/src/main/java/com/fatbook/fatbookapp/core/Ingredient.java
package com.fatbook.fatbookapp.core;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Ingredient implements Serializable {

    private Long pid;

    private String name;

}

File: app/src/main/java/online/fatbook/fatbookapp/core/Ingredient.kt
package online.fatbook.fatbookapp.core

import java.io.Serializable

class Ingredient() : Serializable {
    constructor(pid: Long?, name: String?) : this() {
        this.pid = pid
        this.name = name
    }

    var pid: Long? = null
    var name: String? = null
}